src/mixins/java/org/spongepowered/common/mixin/core/network/ConnectionMixin.java

```diff
@@ -156,13 +155,6 @@ public abstract class ConnectionMixin extends SimpleChannelInboundHandler<Packet
         this.impl$version = new SpongeMinecraftVersion(String.valueOf(version), version);
     }
 
-    @Inject(method = "lambda$doSendPacket$10", at = @At(value = "INVOKE", target = "Lio/netty/util/concurrent/Future;isSuccess()Z"))
```

Comment:
Why was this removed?

Reply:
Moved to `ConnectionMixin_Vanilla` and `ConnectionMixin_Forge` due to injection error.